src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -116,6 +119,8 @@ public partial class DistributedApplication : Microsoft.Extensions.Hosting.IHost
     {
         public DistributedApplication(Microsoft.Extensions.Hosting.IHost host) { }
 
+        public ApplicationModel.ResourceCommandService ResourceCommands { get { throw null; } }
```

Comment:
@JamesNK is there any benefit to making this an interface?

Reply:
Hrm, probably leave it for now. Its not something people will need to mock frequently I guess.

Reply:
I think we'll do it for all 3 services , RNS, RLS, RCS.

Reply:
We could introduce this as a non-breaking change in 9.5 or 10.0. We could introduce the interface, but also still inject the service the way we normally do?